### ISSUE TITLE
Support for LESS or SASS

### ISSUE THREAD

**Issue by Jiri-Kremser** [jbake-org/jbake#223]
It's more a feature request than issue. 

Is [less](http://lesscss.org/) or [sass](http://sass-lang.com/) support on your roadmap? Working with plain css is pretty cumbersome. It can be done manually by a preprocessing, but if it's part of the jbake build it'd be much better because the mvn plugin watch goal will work with it.

**Comment by jonbullock** [comment]
Yes support for them is on the roadmap, potentially as part of the plugin system along with minification of assets.

**Comment by haumacher** [comment]
Is there some information about this "plugin system"? I'd like to integrate a mechanism to obfuscate email addresses from posts and pages (rewriting them as JavaScript that is hard to parse from the page source). I prototyped such post-processing of rendering results in [385130f3a0f6b2c4d1bbdebcb4b58c0ad07f343d] - but of cause, this required hacking the Renderer and is not yet configurable.

**Comment by jonbullock** [comment]
Here's my vision on how it'll work: https://github.com/jbake-org/jbake/pull/78#issuecomment-101652595

**Comment by jonbullock** [comment]
Potential Java solution for SASS support: https://github.com/scireum/server-sass
